pkg/vz/network_darwin.go

```diff
@@ -62,24 +62,9 @@ func forwardPackets(qemuConn *qemuPacketConn, vzConn *packetConn) {
 	defer qemuConn.Close()
 	defer vzConn.Close()
 
-	var wg sync.WaitGroup
-	wg.Add(2)
-
-	go func() {
-		defer wg.Done()
-		if _, err := io.Copy(qemuConn, vzConn); err != nil {
-			logrus.Errorf("Failed to forward packets from VZ to VMNET: %s", err)
```

Comment:
We need to keep this error message, so we get a specific error immediately when it happens.